dockerstack/docker-compose.yml

```
@@ -0,0 +1,20 @@
+version: "3.8"
+services:
+  fileserver:
+    build: ./fileserver
+    ports:
+      - "8081:80"
+    env_file:
+      - vars.env
+    volumes:
+      - ${HOST_CONTENT_ROOT}:/srv/files/download
```

suggestion corresponding to my comment on chaptercard.html

```suggestion
      - ${HOST_CONTENT_ROOT}:/srv/files
```